src/fragments/lib/logging/common/flush_logs/flush_logs.mdx

```diff
@@ -0,0 +1,33 @@
+When using the Amplify Logger, all logged messages are saved locally on the user's device first and then flushed at a set interval that you can [customize](#change-automatic-log-flush-interval). You can also choose to flush events manually if needed by following the steps outlined in this section.
+
+import ios0 from '/src/fragments/lib/logging/ios/flush_logs/flush_logs.mdx';
```

Comment:
It seems like the title for this page is not on the main page but in the fragment? can we change that please?

Also - can we add a quick description here right after the "Manually flush logs" title:
You can choose at anytime to flush the log messages that are saved locally on the user's device, to then send them immediately to Amazon CloudWatch.

Reply:
are you referring to the titles for each section?  There's a main title to the page defined [here](https://github.com/aws-amplify/docs/blob/feature/centralize-logging/src/pages/lib/logging/flush-logs/q/platform/%5Bplatform%5D.mdx#L2C23-L2C23) and then there are two section headers in this common fragment..

Reply:
Added a descriptioned for manually flush logs.  I think the section titles can stay as is since it's in the common fragment and the main page title is defined separately in the main page.

Reply:
I thought the "Manual Refresh" title should also be shown in this page rather than the fragment? I feel like you've done that everywhere else other than this page.

Reply:
See [change_log_levels](https://github.com/aws-amplify/docs/blob/feature/centralize-logging/src/fragments/lib/logging/common/change_log_levels/change_log_levels.mdx) as another example where the common fragment has subsection with titles